testing/trino-testing-services/src/main/java/io/trino/testing/TestOrcWithBloomFilters.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.testing;
+
+public interface TestOrcWithBloomFilters
```

Comment:
This should be an abstract class (BaseOrcWithBloomFiltersTest) that extends from AbstractTestQueryFramework.
Most of the test logic should be in this class with the connector specific derived classes only providing the QueryRunner or any connector specific setup logic. 
See `BaseDynamicPartitionPruningTest` and `TestHiveDynamicPartitionPruningTest` and `TestIcebergDynamicPartitionPruningTest` for an example.

Reply:
Done